README.rst

```diff
@@ -1,34 +1,80 @@
+=========================
 openwisp-network-topology
 =========================
 
 .. image:: https://travis-ci.org/openwisp/openwisp-network-topology.svg?branch=master
-    :target: https://travis-ci.org/openwisp/openwisp-network-topology
+   :target: https://travis-ci.org/openwisp/openwisp-network-topology
+   :alt: CI build status
+
 
 .. image:: https://coveralls.io/repos/github/openwisp/openwisp-network-topology/badge.svg
-    :target: https://coveralls.io/github/openwisp/openwisp-network-topology
+   :target: https://coveralls.io/github/openwisp/openwisp-network-topology
+   :alt: Test Coverage
 
 .. image:: https://requires.io/github/openwisp/openwisp-network-topology/requirements.svg?branch=master
-    :target: https://requires.io/github/openwisp/openwisp-network-topology/requirements/?branch=master
-    :alt: Requirements Status
+   :target: https://requires.io/github/openwisp/openwisp-network-topology/requirements/?branch=master
+   :alt: Requirements Status
 
-.. image:: https://badge.fury.io/py/openwisp-network-topology.svg
-    :target: http://badge.fury.io/py/openwisp-network-topology
+.. image:: https://img.shields.io/gitter/room/nwjs/nw.js.svg
+   :target: https://gitter.im/openwisp/general
+   :alt: chat
 
-------------
+.. image:: https://badge.fury.io/py/openwisp-network-topology.svg
+   :target: http://badge.fury.io/py/openwisp-network-topology
+   :alt: Pypi Version
 
-OpenWISP 2 network topology module (built using Python and Django web-framework).
+.. image:: https://pepy.tech/badge/openwisp-network-topology
+   :target: https://pepy.tech/project/openwisp-network-topology
+   :alt: downloads
 
-.. figure:: docs/images/visualizer.png
-   :align: center
-   :alt:
+.. image:: https://img.shields.io/badge/code%20style-black-000000.svg
+   :target: https://pypi.org/project/black/
+   :alt: code style: black
 
-------------
+.. image:: https://github.com/openwisp/openwisp-network-topology/raw/master/docs/demo_network_topology.gif
+   :alt: Features Highlights
 
```

Comment:
Can you add the openwisp logo as it's done here please?
https://github.com/openwisp/openwisp-utils/blob/master/README.rst

Reply:
Done it but the logo and the git together are not put in other repositories, like django-x509, openwisp-ipam should the logo be added in those as well?

Reply:
Yes, I think a structure that is visually appealing is:

1. GIF
2.short description (you can copy and paste what's in the repo description above or write a better one if you have a better idea).
3 openwisp logo

I think it communicates that the project is being taken care of.
The only missing step in some modules is to release and release more often.
Once we get to release point version (1.0.0, 2.0.0) the process will be complete.

All the steps we are doing are aimed at making each module look something other developers and users can rely on.

Reply:
eg: https://github.com/openwisp/openwisp-network-topology/pull/63/commits/09c52c863ee1eabbd3e12f918f0e61748937274d